### PR TITLE
Adding details about drilling into the Allure results for a failing test

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

File: docs/aries-interop-intro.md [hunk 1]
```diff
@@ -19,69 +19,87 @@ What unites Aries agents are the standards and protocols they aim to adhere to,
...
+   3. Click on the "History" tab to see if it has always failed, or if this is a new issue.
+   4. If it has passed in earlier runs, drill into the passing test to see if you can learn anything from that.
+   5. Dive into the weeds...look at the stack trace, find the associated code, and see if you can figure out what happened. Find anything?  Report it via an issue, or even better, submit a Pull Request to fix the issue. Remember to consider all of the possible places an error could occur (above).
+
+In addition to drilling into a specific test scenario (aka "stories")/case (aka "behavior")/step, you can look at the entire runset history. On the left side menu, click on "Overview", and then take a look at the big "history" graph in the top right, showing how the runset execution has varied over time. Ideally, it's all green, but since you started from a runset that had failures, it won't be. Pretty much every part of the overview page is a drill down link into more and more detailed information about the runset, a specific run of the runset, a specific test case and so on. Lots to look at!
```
Comment (nodlesh) [inline docs/aries-interop-intro.md]: "entire runset history" is not actually the case. It is capped at 20 executions. This was to manage space on the Allure server. We can increase that. We thought 20 was good enough to show some recent history, and we probably don't care about any older than that.

**Review by nodlesh — DISMISSED** [review]
👍

**Comment by swcurran** [comment]
I'll adjust -- thanks for the info.

**Comment by swcurran** [comment]
As I wrote that, I thought that we should check on much storage we are using, so capping it at 20 is a very good idea :-).

**Comment by nodlesh** [comment]
Yes agreed. We could move off the historical records to another server, but it may have minimal value. Right now it is deleted by Allure.

**Review by nodlesh — APPROVED** [review]

**Pull request merged**